test/testdata/array-errors.json

```diff
@@ -3,7 +3,7 @@
 		"lineNumber": "1",
 		"properties": {
 			"recordID": "1",
-			"filler": " "
+			"filler": ""
```

Comment:
@poorgeek The file parser was updated recently to .trim() all fields so I updated the test data to match.